src/encord_active/lib/metrics/heuristic/random.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from encord_active.lib.common.iterator import Iterator
+from encord_active.lib.metrics.metric import (
+    AnnotationType,
+    DataType,
+    Metric,
+    MetricType,
+)
+from encord_active.lib.metrics.writer import CSVMetricWriter
+
+
+class RandomeImageMetric(Metric):
+    def __init__(self):
+        super().__init__(
+            title="Random Values on Images",
+            short_description="Assigns a random value between 0 and 1 to images",
+            long_description="Uses a uniform distribution to generate a value between 0 and 1 to each image",
+            metric_type=MetricType.HEURISTIC,
+            data_type=DataType.IMAGE,
+            annotation_type=[],
+        )
+
+    def execute(self, iterator: Iterator, writer: CSVMetricWriter):
+        valid_annotation_types = {annotation_type.value for annotation_type in self.metadata.annotation_type}
+
+        for _ in iterator.iterate(desc="Progress bar description"):
```

Comment:
[nitpick] more descriptive explanation for the iterator here:

```suggestion
        for _ in iterator.iterate(desc="Assigning random values to images"):
```